tools/c7n_tencentcloud/c7n_tencentcloud/query.py

```diff
@@ -185,9 +190,13 @@ def get_resource_qcs(self, resources):
         # qcs::${ServiceType}:${Region}:${Account}:${ResourcePrefix}/${ResourceId}
         # qcs::cvm:ap-singapore::instance/ins-ibu7wp2a
         qcs_list = []
+
+        # for CAM, there is no region in qcs
+        region = "" if self.resource_type.service in ["cam"] else self.region
```

Comment:
ditto re subclass override.

Reply:
aggred

Reply:
have refactored to use subclass override